tensornetwork/backends/jax/jax_backend.py

```diff
@@ -236,7 +236,8 @@ def eigs(self,
            numeig: int = 6,
            tol: float = 1E-8,
            which: Text = 'LR',
-           maxiter: int = 20) -> Tuple[Tensor, List]:
+           maxiter: int = 20,
+           QR_thresh: float = 1E-12) -> Tuple[Tensor, List]:
```

Comment:
Maybe I just can't see it in this review thingy, but is the meaning of this argument documented? Also, will its not being present in abstract_backend cause issues?

Another concern (with tol as well) is the hard coded default value. In single precision, and certainly in half precision, it's hard to imagine this threshold ever being reached.

Reply:
you're right, I'll fix this